dev-perl/Feature-Compat-Try/Feature-Compat-Try-0.05.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DIST_AUTHOR="PEVANS"
+DIST_VERSION="0.05"
```

Comment:
Do we need this? (Does pkgcheck warn without it?)

See https://wiki.gentoo.org/wiki/Project:Perl/How_to_version-bump_dev-perl#pkgcheck btw.